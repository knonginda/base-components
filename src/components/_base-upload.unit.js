/* eslint-disable no-undef */
import BaseUpload from './_base-upload'

describe('@components/_base-radio', () => {
  it('renders its content', () => {
    const wrapper = shallowMount(BaseUpload)
    wrapper.find('input[type="file"]').trigger('change')
    expect(wrapper).toMatchSnapshot()
  })
})
