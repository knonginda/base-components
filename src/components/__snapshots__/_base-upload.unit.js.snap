// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@components/_base-radio renders its content 1`] = `
<div class="uploader">
  <baseinput-stub type="text" value="" readonly=""></baseinput-stub> <label>
    <div class="selectButton">Select File</div> <input type="file" class="fileButton">
  </label>
</div>
`;
